ci/repo-config/mesosci/cs8/o2-alidist-dataflow.env

```diff
@@ -7,8 +7,5 @@ NO_ASSUME_CONSISTENT_EXTERNALS=
 TRUST_COLLABORATORS=true
 CHECK_NAME=build/O2/alidist-dataflow-cs8
 DONT_USE_COMMENTS=1
-DEVEL_PKGS="$PR_REPO $PR_BRANCH
```

Comment:
You need to have `DEVEL_PKGS="$PR_REPO $PR_BRANCH"`, otherwise it will not test the PR at all.

Reply:
ah okay, fixed